docs/learn-treasury.md

```diff
@@ -2,24 +2,72 @@
+4. Once [Parathreads](learn-parathreads) are live, the per-block auction they have to go through to include their block on the Relay Chain will also likely contribute to the Treasury.
+
+## Creating a Treasury Proposal
+
+The proposer has to deposit 5% of the requested amount or 20 KSM (whichever is higher) as an anti-spam measure. This amount is burned if the proposal is rejected, or refunded otherwise. These values are subject to [governance](learn-governance) so they may change in the future. The way to always find out the up to date values is to [query the constants](https://polkadot.js.org/apps/#/chainstate/constants) as in the image below, or to look at the [source code](https://github.com/paritytech/substrate/blob/master/frame/treasury/src/lib.rs#L784).
+
+![Constants queried in the Polkadot JS UI](/img/treasury/constants.jpg)
+
+### Announcing the Proposal
+
+Before submitting a proposal, it is good practice to announce it. Alternatively, it's good to explain it in detail after it's been proposed.
+
+Several platforms exist for this, none of them official. What matters during an explanation is that the explainer can prove they're behind the proposal, and this is likely easiest to do on a Web3-enabled platform like [Commonwealth](https://commonwealth.im). Commonwealth allows users to log in with their KSM (or Polkadot) address, which immediately binds the identity they're using for on-chain actions with the submissions made on the platform. [Here is an example of an explanatory thread on Commonwealth](https://commonwealth.im/kusama/proposal/discussion/284-treasury-proposal-11-a-demo), made by the same account which created the Treasury spend proposal it's discussing.
+
+Commonwealth is still in beta and can be buggy, so other platforms can be used as well:
+
+- The [Kusama forum](https://forum.kusama.network) and [Polkadot forum](https://forum.polkadot.network) can be used for proposal explanations on Kusama and Polkadot respectively.
+- The [KGP (Kusama Governance Proposals)](https://github.com/kusamanetwork/KGPs) repository is well suited for detailed issues about proposals on Kusama, with the added advantage of easy linkability across repositories and tagging team members.
+- [Riot](https://riot.w3f.tech/#/room/#kusama:matrix.parity.io) is useful, but perhaps too ephemeral for long-form discussions about specific proposals, so it's best suited for promoting any of the above.
+
+Spreading the word about the proposal's explanation is ultimately up to the proposer - the recommended way is using official Riot channels like the [Kusama Direction room](https://riot.w3f.tech/#/room/#kusama:matrix.parity.io) or the [Kusama Watercooler](https://riot.w3f.tech/#/room/#kusamawatercooler:polkadot.builders). For Polkadot, you may want to frequent the [Polkadot Watercooler](https://riot.w3f.tech/#/room/#polkadot-watercooler:matrix.org).
+
+### Creating the Proposal
```

Comment:
This and the following should be in a dedicated tutorial page. This is an example of how to create a proposal with Polkadot JS, but anyone could make a front end to dispatch the appropriate calls.

Reply:
This does not need a separate page imo. We have too many pages as is, and a handy guide right next to the explanation is what most people expect in my experience. Polkadot JS UI is the "most official" platform we have and until there are alternatives that don't involve manually hashing pre-images and sending RPC calls, I would say this should stay in.

Reply:
I won't block it on this, but it should at least not start out in the imperative. At least start with, "One way to create a proposal is to use the Polkadot JS Apps website. From the website, use either ..."

Reply:
Agreed, will amend.